datachannel_test.go

```diff
@@ -157,7 +157,9 @@ func TestDataChannel_MessagesAreOrdered(t *testing.T) {
 	inner := func(msg DataChannelMessage) {
 		// randomly sleep
 		// NB: The big.Int/crypto.Rand is overkill but makes the linter happy
```

Comment:
i would change this to read `The big.Int/crypto.Rand is overkill, so disable that rule with #nosec` or something like that

(i had to look up what #nosec means)